contracts/eth/nearprover/.catalog-info.yaml

```
@@ -0,0 +1,22 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: prover-on-ethereum
+  title: "Prover on Ethereum"
+  description: |-
+    Takes proof of a Receipt from NEAR blockchain and check it is valid and included on chain.
+  tags:
+    - contract
+    - ethereum
+  links: []
+  annotations:
+    aurora.dev/security-tier: "1"
+spec:
+  owner: bridge-team
+  type: contract
+  lifecycle: production
+  system: bridge-protocol
+  deployedAt:
+    - contract:ethereum/mainnet/0x051ad3f020274910065dcb421629cd2e6e5b46c4
+  interactsWith: []
```

Prover interacts with NEAR light client on Ethereum

```suggestion
  interactsWith:
    - contract:ethereum/mainnet/0x3be7df8db39996a837041bb8ee0dadf60f767038
```